emails/views.py

```diff
@@ -609,6 +609,7 @@ def _get_domain_address(local_portion, domain_portion):
                 domain_address = DomainAddress.make_domain_address(
                     locked_profile, local_portion, True
                 )
+            domain_address.num_forwarded += 1
```

Comment:
After writing a test for this, I don't think this is causing the original bug? I got this in the new test:
```
    @patch('emails.views.ses_relay_email')
    def test_domain_recipient(self, mock_ses_relay_email):
        import ipdb; ipdb.set_trace()
        _sns_notification(EMAIL_SNS_BODIES['domain_recipient'])

        mock_ses_relay_email.assert_called_once()
        da = DomainAddress.objects.get(
            user=self.premium_user, address='wildcard'
        )
>       assert da.num_forwarded == 1
E       AssertionError: assert 2 == 1
E        +  where 2 = <DomainAddress: wildcard>.num_forwarded
```

Let me add my test changes to this PR ... sorry this PR might become more of a "wrap this in tests until we find the bug" PR.